server/app/util/db.js

```diff
@@ -0,0 +1,42 @@
+import mongoose from 'mongoose';
+import logger from './logger';
+
+export const connectDb = dbUrl => {
+  mongoose.connect(dbUrl);
+
+  // Mongo connection events
+  mongoose.connection.on('connected', () => {
+    logger.info(`💾  Mongo connected to ${dbUrl}`);
```

Comment:
I can take it out. Just wanted something that stands out to the eye of when db events occur.